.github/workflows/release.yaml

```diff
@@ -56,7 +55,9 @@ jobs:
           draft: true
           prerelease: true
       - name: Upload cloud-hypervisor
-        if: github.event_name == 'create' && github.event.ref_type == 'tag'
+        if: |
+          startsWith(matrix.platform.target, 'x86_64-unknown-linux-gnu') &&
+          github.event_name == 'create' && github.event.ref_type == 'tag'
```

Comment:
I believe a cleaner syntax would be:

`if: ${{ github.event_name == 'create' && github.event.ref_type == 'tag' && matrix.platform.target = 'x86_64-unknown-linux-gnu'}}`

Reply:
done